strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/GraphExporter.java

```diff
@@ -0,0 +1,21 @@
+package org.arquillian.smart.testing.strategies.affected;
+
+import org.jgrapht.DirectedGraph;
+import org.jgrapht.ext.DOTExporter;
+import org.jgrapht.ext.ExportException;
+import org.jgrapht.graph.DefaultEdge;
+
+import java.io.File;
+
+class GraphExporter
+{
+    public void dumpGraph(DirectedGraph<JavaElement, DefaultEdge> graph, String graphName) {
+        try {
+            new DOTExporter<JavaElement, DefaultEdge>(component -> "\"" + component.getClassName() + "\"",
+                null, null)
+                .exportGraph(graph, new File(System.getProperty("java.io.tmpdir") + "/graph-" + graphName + ".dot"));
```

Comment:
Log also the information that it has been exported and the location of the file

Reply:
wouldn't make sense to store the file inside of `target/smart-testing` directory of the particular module/project? I guess, that this directory will contain all ST relevant information/files

Reply:
yeah